test-infra/camel-test-infra-openldap/src/test/java/org/apache/camel/test/infra/openldap/common/OpenldapProperties.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.camel.test.infra.openldap.common;
+
+public final class OpenldapProperties {
+    public static final String HOST = "openldap.host";
+    public static final String PORT_389 = "openldap.port.389";
+    public static final String PORT_636 = "openldap.port.636";
```

Comment:
For clarity, maybe use `PORT_LDAP` and `PORT_LDAP_OVER_SSL` along with `openldap.port.ldap` and `openldap.port.ldap.ssl` respectively? I had to google what these ports are ... so, it would make it easier to figure it out in the future.

Reply:
you are right, when I started I had to google it too, but forgot to change there, will do